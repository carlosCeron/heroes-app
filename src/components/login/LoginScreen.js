import React from 'react'
import PropTypes from 'prop-types'

const LoginScreen = props => {

  console.log('Login Screen');

  return (
    <div>
      <h1>Login Screen</h1>
    </div>
  )
}

LoginScreen.propTypes = {

}

export default LoginScreen
