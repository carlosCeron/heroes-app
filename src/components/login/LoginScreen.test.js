import React from 'react';
import { shallow } from "enzyme";
import LoginScreen from "./LoginScreen";


describe('Debe renderizar el componente login', () => {

  test('Evalua el componente LoginScreen', () => {

    /*let wrapper = shallow(<LoginScreen />);

    expect(wrapper).toMatchSnapshot(); */

  });

});