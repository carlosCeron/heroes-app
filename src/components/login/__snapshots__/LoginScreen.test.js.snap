// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Debe renderizar el componente login Evalua el componente LoginScreen 1`] = `
<div>
  <h1>
    Login Screen
  </h1>
</div>
`;
